lib/task-data/tasks/delete-raid.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015, EMC, Inc.
+
+module.exports = {
+    friendlyName: 'Delete RAID via Storcli',
+    injectableName: 'Task.Raid.Delete',
+    implementsTask: 'Task.Base.Linux.Commands',
+    options: {
+        deleteAll: true,
+        controller: 0,
+        raidIds: [], /*[
+            {
+                id: 0
+            },
+            {
+                id: 1
+            }
+        ],*/
+        path: '/opt/MegaRAID/storcli/storcli64',
+        commands: [
+            '{{#options.deleteAll}}\
+				sudo {{options.path}} /c{{options.controller}}/vall del force\
+			{{/options.deleteAll}}\
+			{{^options.deleteAll}}\
+			{{#options.raidIds}}\
+				sudo {{options.path}} /c{{options.controller}}/v{{id}} del force;\
```

Comment:
@WangWinson , we can use the {{.}} to represent the current array element, so we don't have to change the raidIds  to something like [ {id:0}, {id:1}], we can use the simper one [0, 1]